atom/browser/atom_browser_client.cc

```diff
@@ -608,11 +608,6 @@ bool AtomBrowserClient::CanCreateWindow(
 
   int opener_render_process_id = opener->GetProcess()->GetID();
 
-  if (IsRendererSandboxed(opener_render_process_id)) {
```

Comment:
Why did this go away?

Reply:
@MarshallOfSound it's now covered by `RendererUsesNativeWindowOpen`

Reply:
`sandbox: true` also sets `nativeWindowOpen: true` in `webPreferences`

otherwise, this would have to be
```
  if (IsRendererSandboxed(opener_render_process_id) || RendererUsesNativeWindowOpen(opener_render_process_id)) {
    if (RendererDisablesPopups(opener_render_process_id)) {
      // <webview> without allowpopups attribute should return
      // null from window.open calls
      return false;
    } else {
      *no_javascript_access = false;
      return true;
    }
  }
```

Reply:
Sweet thanks, didn't trace that, good to have the context 👍